Technical-Steering-Committee/issues.md

```diff
@@ -0,0 +1,49 @@
+# Issues
+
+All the squads within the Zowe projects use GitHub to track the issues raised against the projects
+they are responsible for. The issues could be questions, bug reports, request for enhancements and others.
+Every squad responds to the new issues within the 90 days of the creation of the issue. 
+
+The squads can specify the details of the issue processing for specific project within the project
+repository in the ISSUES.md file in the root of the repository. 
+
+Every squad is expected to have at least one weekly meeting. The meeting should among other responsibilities
+triage the new issues and decide what to do with the issues. 
+
+The triage of the issues is done via attaching the labels to the issues. The recommended labels to use are:
```

Comment:
Rewording suggestion:

"The triaging of an issue is done by attaching labels to it (see below for the recommended set of labels):"